packages/terra-graphs/src/components/Line/LineGraph.jsx

```diff
@@ -17,17 +21,34 @@ const propTypes = {
    * Data to be displayed in graph.
    */
   dataset: PropTypes.arrayOf(PropTypes.object),
+  /**
+   * dynamic data to update the initial data
+   */
+  panData: PropTypes.object,
```

Comment:
Does this need to be a separate prop? I notice the shape is different than the dataset prop, but I'm not sure why that is. It seems like it would be more straightforward to have a single data prop and flex how the component handles changes to it (reflow or not) based on the component's internal lifecycle.

Reply:
For some unknown reason, the shape of the new panning dataset is different from the initial dataset and this has been baked into the reflow logic by another contributor. Ideally the format of the initial and panning datasets should've been the same. But since the logic is what it is for now (we'll revise in the upcoming major update project), i used a second prop as a work around. But I can try again to see if we can use a single prop for this component.